component/vshn_postgres.jsonnet

```diff
@@ -290,6 +360,38 @@ local composition =
             comp.FromCompositeFieldPathWithTransformSuffix('metadata.labels[crossplane.io/claim-name]', 'spec.writeConnectionSecretToRef.name', 'connection'),
           ],
         },
+        // s3 bucket creation for backup purposes
+        {
+          base: objectBucket,
```

Comment:
The `s3-bucket` suffixes and pre-fixes seem somewhat redundant here. You create an `ObjectBucket` object, so it's clear what it's doing. The `s3-bucket` suffix doesn't provide more information here. 

Also, we'll end up with buckets that start with `s3-bucket-*`, which is also not very helpful when browsing the buckets.

Reply:
And something else:

Please use the `XObjectBucket` (composite) and not the `ObjectBucket` (claim). The claims are not intended to be used in compositions.

The `XObjectBucket` behaves like any other Crossplane managed resource and is cluster scoped. So, you can then also remove the namespace as well.